apps/quarkus-spöklik-encoding/threshold.conf

```diff
@@ -20,4 +20,7 @@ linux.container.time.to.first.ok.request.threshold.ms=1500
 linux.container.RSS.threshold.kB=55000
 @IfQuarkusVersion(min ="3.7.0")
 linux.container.executable.size.threshold.kB=51313
-
+@IfQuarkusVersion(min ="3.8.6")
+linux.container.executable.size.threshold.kB=51991
+@IfQuarkusVersion(min ="3.15.1")
+linux.container.executable.size.threshold.kB=52860
```

Comment:
How are these related to the issue at hand?

Reply:
Totally unrelated, I was just stabilizing the run as I was testing it. Gonna amend the PR to note it.